spring-cloud-azure-samples/spring-cloud-azure-storage-queue-integration-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.cloud.azure.credentialFilePath=[credential-file-path]
```

Comment:
I have changed property format. Now all become something like `credential-file-path` instead of `credentialFilePath`